app.go

```diff
@@ -4,8 +4,10 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var Environment string
-var broadcaster Broadcaster
+var (
+	Environment string
```

Comment:
Not part of the scope of my PR but thought it would be nice to place them in one var declaration.